client/src/core/PanelGraph.js

```diff
@@ -183,6 +182,14 @@ class PanelGraph {
     }
   }
 
+  get_glossary_keys(){
+    if (this.glossary_keys === false){
+      return [];
+    } else {
+      return this.glossary_keys;
+    }
+  }
```

Comment:
Did the ternary operator not work for this one? like `this.glossary_keys ? this.glossary_keys : []`

Reply:
Or `this.glossary_keys || []`